### ISSUE TITLE
WelcomeView doesn't appear

### ISSUE THREAD

**Issue by NathanBnm** [manexim/home#28]
Here is the screen I have when I start the app:
![image](https://user-images.githubusercontent.com/45366162/62531408-aeb59980-b842-11e9-9d16-30880dcc7c69.png)

**Comment by meisenzahl** [comment]
Thanks for creating the issue. I'll take care of it.

**Comment by meisenzahl** [comment]
Did you start the app for the first time?

It looks like you have started the app before. Then the WelcomeView is no longer displayed.

If no devices are found in your network, then the page is empty as in your case. But I will add a page that shows that no devices were found.

**Comment by meisenzahl** [comment]
A message is now displayed that devices are being looked for in the network. Version 0.4.1 has already been submitted to the AppCenter for review.

![](https://user-images.githubusercontent.com/10796736/62596361-45856300-b8e2-11e9-87e8-6430dac3e7a8.png)

**Comment by NathanBnm** [comment]
> Did you start the app for the first time?
> 
Yes but I tried to uninstall and reinstall it again, and I still don't have the Welcome Screen displayed.
>
> It looks like you have started the app before. Then the WelcomeView is no longer displayed.
> 
> If no devices are found in your network, then the page is empty as in your case. But I will add a page that shows that no devices were found.
>
Great :+1:

**Comment by NathanBnm** [comment]
@meisenzahl maybe if there is no available device you should display a different Welcome View with "No devices found" and different options maybe "Rescan" and "Add a device" idk.

**Comment by meisenzahl** [comment]
@NathanBnm does version 0.4.1 show you the message **Looking for smart home gadgets to control**?

Home is constantly looking for devices, so I think the message with the spinner fits.

**Comment by NathanBnm** [comment]
I'll try that.

I think it would be great to add a timer. I mean you can search for devices maybe during 1 minute and then display a "No devices found" if no device was found and display different options. I'm thinking about "Search for devices" and "Get help".

It would be a good idea because if users have a device that is not detected, they can figure out that the app didn't find it and try to search again and get help.

Maybe you can link the "Get help" option to a web page or a dialog with some tips to help device's connection like "Make sure your device is compliant (list of supported models)" or "Make sure your device is not to far"...

What do you think?

**Comment by NathanBnm** [comment]
> @NathanBnm does version 0.4.1 show you the message **Looking for smart home gadgets to control**?
> 

I've got this message, but as I have no devices so it's an infinite loop.
